controllers/suite_test.go

```diff
@@ -56,10 +62,15 @@ var _ = BeforeSuite(func() {
 	var ctx context.Context
 	ctx, cancel = context.WithCancel(context.TODO())
 
+	testScheme := scheme.Scheme
+	err := doguv1.AddToScheme(testScheme)
+	Expect(err).NotTo(HaveOccurred())
+
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
-		CRDDirectoryPaths:     []string{filepath.Join("..", "config", "bases")},
+		CRDDirectoryPaths:     []string{filepath.Join("..", "vendor", "github.com", "cloudogu", "k8s-dogu-operator", "api", "v1")},
 		ErrorIfCRDPathMissing: false,
```

Comment:
Maybe we should set this to true? `ErrorIfCRDPathMissing`. Sounds reasonable to me.